Cargo.toml

```diff
@@ -24,7 +24,9 @@ exclude = [".cargo/**/*", ".github/**/*"]
 all-features = true
 
 [dependencies]
+proptest = { version = "1.0.0", optional = true }
 serde = { version = "1", features = ["derive"], optional = true }
 
 [features]
 serde1 = ["serde"]
+proptest = ["dep:proptest"]
```

Comment:
This won't work with older versions of Cargo, which camino supports. I'd recommend calling the feature `proptest1` to match `serde1`.

Reply:
Done